File: backend/booking-service/src/main/java/com/project/booking/dto/passenger/PassengerResponse.java
package com.project.booking.dto.passenger;

import com.project.booking.model.enums.DocumentType;
import java.time.LocalDate;
import java.time.LocalDateTime;

public record PassengerResponse(
        Long id,
        Long userId,
        String fullName,
        LocalDate dateOfBirth,
        String gender,
        String phoneNumber,
        String email,
        DocumentType idCardType,
        String identificationNumber,
        String documentNote,
        String idCardImageUrl,
        LocalDateTime createdAt,
        LocalDateTime updatedAt) {
}

File: frontend-android/app/src/main/java/com/project/cruise/android/data/dto/booking/PassengerResponse.kt
package com.project.cruise.android.data.dto.booking

data class PassengerResponse(
    val id: Long?,
    val fullName: String?,
    val dateOfBirth: String?,
    val gender: String?,
    val phoneNumber: String?,
    val email: String?,
    val idCardType: DocumentType?,
    val identificationNumber: String?,
    val documentNote: String?,
    val idCardImageUrl: String?,
    val createdAt: String?,
    val updatedAt: String?
)